app/views/checkout/_terms_and_conditions.html.haml

```diff
@@ -0,0 +1,2 @@
+%p
+  = t('.message_html', terms_and_conditions_link: link_to( t( '.link_text' ), current_order.distributor.terms_and_conditions.url, target: '_blank')) if current_order.distributor.terms_and_conditions.present?
```

Comment:
Uh, this is a long line, doesn't fit in the Github view.

Anyway, I read that `present?` always returns true here. You need to use `file?`. Don't use `exists?` because that queries Amazon to see if it's really there.

Reply:
:+1: done